webapp/TargetedMS/js/QCTrendPlotPanel.js

```diff
@@ -1268,6 +1290,15 @@ Ext4.define('LABKEY.targetedms.QCTrendPlotPanel', {
             metricProps = this.getMetricPropsById(this.metric),
             me = this;
 
+        let panelY = me.canUserEdit() ? -375 : -270;
+        if (valueName === "TrailingMean" || valueName === "TrailingCV") {
+            panelY = panelY + 200;
+        }
+
+        let trailingRuns = this.trailingRuns;
+        let trailingStartDate = row['TrailingStartDate'];
+        let trailingEndDate = row['TrailingEndDate'];
```

Comment:
See my earlier comment - I bet this could just use the "regular" date for the row.